Issue #34: News Feed Page

cottonswab opened the pull request:
- Create "feed" component
- Send fetch request to get most recent photo from each user followed (backend api loop (get user.following.photos.photo_n))
- Front end display results

cottonswab committed:
Issue #34: News Feed Page

- Create "feed" component
- Send fetch request to get most recent photo from each user followed (backend api loop (get user.following.photos.photo_n))
- Front end display results

loganmcmullen commented on client/src/pages/homepage.js:
```diff
@@ -186,11 +212,11 @@ class HomePage extends Component {
   //             }}
```
delete this commented out stuff of use /* ------- */

loganmcmullen commented on client/src/pages/homepage.js:
```diff
@@ -186,11 +212,11 @@ class HomePage extends Component {
   //       </Row>
   //     </Container>
-
+/*
```
remove some of the commented stuff

cottonswab replied:
Will do.

loganmcmullen commented on client/src/components/displayHomepage.js:
```diff
@@ -0,0 +1,103 @@
+        return (
+            <Container>
+                <Row>
+                    {this.state.img.map(item => {
```
I see you used the template from the profile pictures which is great. Good way to save time and keep consistency throughout

gregtuc reviewed (changes requested):
There needs to be an ability to see who posted the photo. I recommend making another state variable and adding the name of the user who's photo it is to the Card. It would also be better if the cards were laid out vertically in a single column, not in Profile-page format. After this is done the request can be pulled in

ginywiny commented on client/src/components/displayHomepage.js:
```diff
@@ -0,0 +1,103 @@
+                                    />
+                                    <Card.Body>
+                                        <Card.Title>{item.description}</Card.Title>
+                                        <Card.Text>This will be the comment section</Card.Text>
```
These sections will be updated with the comments section that I am still in the process of completing.

cottonswab committed:
Issue #34: News Feed Page

- Create "feed"
- Send fetch request to get most recent photo from each user followed (backend api loop (get user.following.photos.photo_n))
- Front end display results

gregtuc reviewed (approved):
Perfect. Thanks for making the changes. Looks good
